### ISSUE TITLE
Typedesc not available in function symbol

### ISSUE THREAD

**Issue by pubudu91** [ballerina-platform/ballerina-lang#26230]
When building the public symbol for a function, the typedesc is not built. Hence it's not available to the user.

**Comment by rasika** [comment]
Typedesc is not available even without `public` qualifier.

```
public function main() {
    foo(true, 1.0);
}

function foo(boolean a, float b) returns int {
    return 0;
}
```

**Comment by pubudu91** [comment]
The above issue you mentioned is for the return type we discussed right?

**Comment by pubudu91** [comment]
Closing since this was fixed with #26232
